_docs/geyser/custom-blocks.md

```diff
@@ -0,0 +1,473 @@
+---
+layout: page
```

Comment:
layout and permalink can be removed, they're no longer used
```suggestion
```

Reply:
https://github.com/GeyserMC/GeyserWiki/pull/115/commits/9066dbd42a7d8e8f9fe2e3a0ed5f7f2c993c6cec#diff-6907c07388b3f34cb8c388adc86d69766e7681b013ea0bf58fe99988fe657ee7L2